numba/np/ufunc/_internal.c

```diff
@@ -329,6 +330,20 @@ init_ufunc_dispatch(void)
             result = -1; /* Unknown method */
         }
         if (result < 0) break;
+
+#if PY_VERSION_HEX >= 0x03070000
```

Comment:
I suppose you probably only support >=3.7 and the `#ifdef`'s are actually unnecessary?

Reply:
Numba supports >= 3.7, so no need to guard.